packages/buidler-core/src/internal/core/config/config-validation.ts

```diff
@@ -80,14 +80,31 @@ const BuidlerNetworkAccount = t.type({
   balance: t.string,
 });
 
+const HDAccountsConfig = t.type({
+  mnemonic: t.string,
+  initialIndex: optional(t.number),
+  count: optional(t.number),
+  path: optional(t.string),
+});
+
+const BuidlerNetworkHDAccountsConfig = t.type({
```

Comment:
I think you can replace this with:

```
const BuidlerNetworkHDAccountsConfig = t.intersection([HDAccountsConfig, { accountsBalance: optional(t.string)}])
```

Reply:
Nice!

Reply:
But now that I think of, that will result in really cryptic error messages.